internal/pkg/deploy/cloudformation/stack/backend_svc_test.go

```diff
@@ -273,9 +273,11 @@ Outputs:
 			},
 			Manifest: mft,
 			RuntimeConfig: RuntimeConfig{
-				Image: &ECRImage{
-					RepoURL:  testImageRepoURL,
-					ImageTag: testImageTag,
+				Images: map[string]ECRImage{
```

Comment:
Do we have any test cases with more than one entry in the map? to make sure we are translating multiple container images

Reply:
As of now we are passing only the main container config. In the upcoming PR I am adding sidecars as well, in that case I will update the testcases. Does that sound good?

Reply:
sounds great!